src/styles/styles.js

```diff
@@ -2217,21 +2217,18 @@ const styles = {
         textAlign: 'center',
     },
 
-    twoFactorAuthCodesButtonsContainer: {
+    twoFactorAuthCodesButtonsContainer: (isExtraSmallScreenWidth) => ({
         flexDirection: 'row',
+        flexWrap: isExtraSmallScreenWidth ? 'wrap' : 'nowrap',
```

Comment:
>Why do we need this condition? We can use wrap by default. If there is insufficient space, it will be wrapped otherwise not.

Yes, but then it also wraps in other screen sizes such as iPhone SE, 7 or 8, which does not comply with the designs. This is a fix made specifically for screens with width < 320, such as the Galaxy Fold